src/stores/compliance-ccpa-guide.md

```diff
@@ -0,0 +1,145 @@
+
+  1.  Identify the parties with whom you share data. Your list will include service providers and third-parties such as advertising networks, internet service providers, data analytics providers, government entities, operating systems and platforms, social networks, and consumer data resellers who do not directly collect personal information from your customers.
+
+      **Service Providers**:  Those who have access to your customer data for a business purpose, and provide services on your behalf. For example, Adobe is service provider, as are some developers of customizations and extensions and services. 
+
+      Check the default settings of Google Universal Analytics, Google Tag Manager &mdash; and any other data services you use &mdash; and make any changes necessary to comply with the regulation. To learn more, see [Google Analytics Settings for GDPR]({% link stores/compliance-gdpr-google.md -%}).  
+
+      **Other Third Parties**: Those with whom you share or sell customer data. For example, you might share customer data with an advertising network in exchange for advertising.
+
+### Step 3: Map the customer journey and data collection process in your store(s).
+
+**Stakeholders:** User Experience, Information Technology, Administrative Support
+
+1.  Identify each point in the [customer journey]({% link getting-started/quick-tour.md -%}) where personal information is collected, and the type of information that is collected at each step.
+
+    Visitors to your site must be notified in advance, or at the point of data collection. For example, a Magento store without custom integrations collects personal information when a customer account is created, and during checkout. If your store has custom integrations, there might be additional data items and attributes to identify. 
+
+1.  Although the following topics were written to help Magento merchants in the EU comply with the GDPR, they can be adapted for CCPA compliance. For data flow diagrams and technical information about database entities, see the topic For your version of Magento.
+
+      - [General Data Protection Regulation][2] (Magento 2.x)
+      - [General Data Protection Regulation][3] (Magento 1.x)
+
+    ![]({% link images/images/gdpr-frontend-data-entry-points.png %}){: .zoom}
+    _Storefront Data Entry Points_
+
+### Step 4: Establish procedures and mechanisms to respond to customer requests.
+
+**Stakeholders:** Customer Service, Information Technology, User Experience, Administrative Support
+
+1.  Verify the identity of people who request to know, opt-out, or delete, regardless of whether they have a password-protected customer account, or are visiting your store as a guest.
+
+1.  Respond to requests to know, opt out, and delete within a specific (as yet undefined) time frame. 
```

Comment:
Does "opt out" need to be hyphenated if it's not modifying a noun? I see a mixed use of hyphenation and non-hyphenation for both cases.

Reply:
Opt out is not hyphenated as a verb; opt-in is hyphenated as a noun. It's correct as is.